### DISCUSSION TITLE
Streetname "sign" - for the visual impaired

### DISCUSSION THREAD

**Discussion by gh-ix** [andreynovikov/trekarta#268] [Ideas]
Hi, huge THANKS!  trekarta is really well designed and a wonderful app for me to record my bicycle journeys - mostly always the same few (work, lake, friends).  So main purpose for me is kind of a power-meter - and to optimize tracks.  I prefer beautiful routes over shortest route.  For strict navigation, I use MagicEarth, but **after arrival/returning, I analyze my route to further avoid useless detours with trekarta.  Unfortunately the street names are much too hard to read for my old eyes**.
I can imagine that dynamic street name scaling would be a huge task to implement.  Maybe a static switch/user-setting to increase font size for street names only?
To my imagination, the perfect solution would be a **imposed 'streetname-sign' in one of the corners** (user-selectable), **showing the name of the centered street in really good readable font size!**  That way, map visualization doesn't get worse and I thin it's also much easier to implement.  If live position is activated, the name of the street I'm currently traveling along, if position tracking is off (map reading mode), some kind of crosshair should indicate the target.

EDIT: I missed your geo location - apparent reason why PayPal Donations are not possible these days...  sorry.  Still I'd like to support your great work.  Guess wire transfer is out of scope too... I don't use Google PlayStore (i'm on GrapheneOS). so I can't even leave a rating (and wouldn't be able to purchase if it's commercial).

**Comment by andreynovikov** [comment]
Yes, transferring money is a little bit difficult nowadays.

As for your request - getting current road name is a bit tricky as for Trekarta it is just a styled line, nothing more. I hope to add true routing some day, then information about roads will be available.
